src/BezierCurve.cpp

```diff
@@ -113,6 +166,9 @@ std::vector<double> BezierCurve::findTimeForY(double Y)
 		A = points[0].y - 2*points[1].y + points[2].y;
 		B = -2*points[0].y + 2*points[1].y;
 		C = points[0].y - Y;
+		A = MathExt::roundToDecimal(A, 10);
+		B = MathExt::roundToDecimal(B, 10);
+		C = MathExt::roundToDecimal(C, 10);
```

Comment:
Done this way to prevent errors. Can end up with an unsolvable equation if rounding is not kept in check. Review Later for accuracy and better error checking.